### PR TITLE
tag more tests with ignoreNotImplementedSelectors: pragma

### PR DESCRIPTION
**Pull request by MarcusDenker**
... slowly...

### CODE REVIEW

File: src/Deprecated12/SHRBTextStyler.extension.st [hunk 1]
```diff
@@ -19,12 +19,3 @@ SHRBTextStyler class >> initialTextAttributesForPixelHeight: aNumber [
...
 ]
-
-{ #category : #'*Deprecated12' }
-SHRBTextStyler class >> initialTextAttributesForPixelHeight: aNumber styleTable: aStyleTable [
-
-	self
-		deprecated: 'Use #initialTextAttributesForStyleTable: instead.'
-		transformWith: '`@rcv initialTextAttributesForPixelHeight: `@arg1 styleTable: `arg2' -> '`@rcv initialTextAttributesForStyleTable: `@arg2'.
-	^ self initialTextAttributesForStyleTable: aStyleTable
-]
```
Comment (jecisc) [inline src/Deprecated12/SHRBTextStyler.extension.st:30]: Is this removal intentional?

File: src/TraitsV2-Tests/ClassTraitTest.class.st [hunk 2]
```diff
@@ -50,6 +50,7 @@ ClassTraitTest >> testConflictsAliasesAndExclusions [
 { #category : #tests }
 ClassTraitTest >> testInitialization [
 	| classTrait |
+		<ignoreNotImplementedSelectors: #(m2ClassSide:)>
```
Comment (jecisc) [inline src/TraitsV2-Tests/ClassTraitTest.class.st]:
```suggestion
	<ignoreNotImplementedSelectors: #(m2ClassSide:)>
```

File: src/Deprecated12/SHRBTextStyler.extension.st (hunk 1 above)
Comment (MarcusDenker) [reply src/Deprecated12/SHRBTextStyler.extension.st:30]: no, and I guess I have to close this PR again as I have no clue how to fix it, as the image I used I did a spec pr before and spec thinks that it owns this method

Comment (MarcusDenker) [reply src/Deprecated12/SHRBTextStyler.extension.st:30]: I opened:

SHRBTextStyler class>>#initialTextAttributesForPixelHeight: is in Deprecated12, but spec thinks it is in spec #13990

Comment (MarcusDenker) [reply src/Deprecated12/SHRBTextStyler.extension.st:30]: So how do I fix this PR? I do not want to redo that trivial copy and past yet again...

Comment (MarcusDenker) [reply src/Deprecated12/SHRBTextStyler.extension.st:30]: In my image, this method is in Deprecated12, I can even edit it, but when I commit it is ingored

Comment (jecisc) [reply src/Deprecated12/SHRBTextStyler.extension.st:30]: https://stackoverflow.com/questions/215718/how-do-i-reset-or-revert-a-file-to-a-specific-revision

This seems to be what you need

Comment (MarcusDenker) [reply src/Deprecated12/SHRBTextStyler.extension.st:30]: that's too complex for me (and will nor work, as there is no verson in git as iceberg just does not see this change)
